docs/Docker.md

```diff
@@ -0,0 +1,53 @@
+# Docker
+
+You may use docker images of the ADBench project without having to install dependencies on your PC.
+
+## Building docker image
+
+1. Clone repository
+2. Add your AD tools if nessasary
+3. Run `docker build -f <dockerfile name> -t <docker image name> .` from root directory of ADBench project  
```

Comment:
Missed articles: `the root directory`, `of the ADBench project`

Reply:
fixed